src/l10n/jp-JP.js

```diff
@@ -185,7 +186,7 @@ const l10n = {
     repeatPasswordInput: {
       label: 'パスワードをもう一度入力してください。',
       errors: {
-        passwordsDoNotMatch: 'パスワードが間違っています。',
+        passwordsDoNotMatch: '2つのパスワードが違います。',
```

Comment:
@Leo-Emurgo is this 2 in the beginning on purpose?

Reply:
Yes. I will be making a lot of edits this afternoon, though. So please don't apply any changes until tonight :)